deltachat/rpc.go

```diff
@@ -28,7 +32,7 @@ type Event struct {
 	StatusUpdateSerial uint
 }
 
-type _Params struct {
+type notifyParams struct {
```

Comment:
If this is a private struct to RpcIO, it should be in lowercase, if it's a public struct (for other Rpc implementations) I suggest just renaming it to something not underscore.

Reply:
I agree this name is better than _Params (which is also private btw, underscore is equivalent to starting with lower case) but this is unrelated to this PR, I think it is simple enough to leave but please in the future lets try to only modify what it is really needed for the PR at hand, creating new "code cleanup" PRs is cheap :D